test/cache.test.js

```diff
@@ -376,7 +376,7 @@ test.cb("should allow to specify the .babelrc file", t => {
 
     fs.readdir(t.context.cacheDirectory, (err, files) => {
       t.is(err, null);
-      t.true(files.length === 2);
+      t.true(files.length === 1);
```

Comment:
This changes because the cached files now have an identical key.

Reply:
Well, it's good that we can actually see the improvement caused by this change :smile: